test-cases/features/repair-based-operations/longevity-cdc-100gb-4h-RemoveNode.yaml

```diff
@@ -0,0 +1,6 @@
+nemesis_class_name: 'TerminateAndRemoveNodeMonkey'
+nemesis_interval: 5
+nemesis_during_prepare: false
```

Comment:
Please, add `seeds_num` or chande `nemesis_filter_seeds` to True
"longevity-cdc-100gb-4h.yaml" runs with 1 seed.

Reply:
Same as above.